specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2022-05-31/digitaltwins.json

```diff
@@ -0,0 +1,1622 @@
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        },
+        "innererror": {
+          "description": "An object containing more specific information than the current object about the error.",
+          "$ref": "#/definitions/InnerError"
+        }
+      }
+    },
+    "InnerError": {
+      "description": "A more specific error description than was provided by the containing error.",
+      "properties": {
+        "code": {
+          "description": "A more specific error code than was provided by the containing error.",
+          "type": "string"
+        },
+        "innererror": {
+          "description": "An object containing more specific information than the current object about the error.",
+          "$ref": "#/definitions/InnerError"
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+    "oauth2": {
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize?resource=0b07f429-9f4b-4714-9392-cc5e8e80c8b0",
+      "type": "oauth2"
+    }
+  },
+  "parameters": {
+    "eventRouteId": {
+      "name": "id",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The id for an event route. The id is unique within event routes and case sensitive.",
+      "x-ms-parameter-location": "method"
+    },
+    "includeModelDefinition": {
+      "name": "includeModelDefinition",
+      "description": "When true the model definition will be returned as part of the result.",
+      "in": "query",
+      "required": false,
+      "type": "boolean",
+      "default": false,
+      "x-ms-parameter-location": "method"
+    },
+    "digitalTwinId": {
+      "name": "id",
+      "in": "path",
+      "description": "The id of the digital twin. The id is unique within the service and case sensitive.",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "modelId": {
+      "name": "id",
+      "in": "path",
+      "description": "The id for the model. The id is globally unique and case sensitive.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "relationshipId": {
+      "name": "relationshipId",
+      "description": "The id of the relationship. The id is unique within the digital twin and case sensitive.",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "componentPath": {
+      "name": "componentPath",
+      "in": "path",
+      "description": "The name of the DTDL component.",
+      "type": "string",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "api-version": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The requested API version.",
+      "required": true,
+      "type": "string",
+      "minLength": 10
```

Comment:
You might just use minLength=1
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L636

Reply:
Done